Example/CustomActionControllers/Youtube.swift

```diff
@@ -91,16 +91,20 @@ public class YoutubeActionController: ActionController<YoutubeCell, ActionData,
         settings.animation.present.duration = 0.6
         settings.animation.dismiss.duration = 0.6
         settings.animation.dismiss.offset = 30
-        settings.animation.dismiss.options = .CurveLinear
+        settings.animation.dismiss.options = .curveLinear
         
-        cellSpec = .NibFile(nibName: "YoutubeCell", bundle: NSBundle(forClass: YoutubeCell.self), height: { _  in 46 })
+        cellSpec = .nibFile(nibName: "YoutubeCell", bundle: Bundle(for: YoutubeCell.self), height: { _  in 46 })
         
         onConfigureCellForAction = { cell, action, indexPath in
             cell.setup(action.data?.title, detail: action.data?.subtitle, image: action.data?.image)
             cell.alpha = action.enabled ? 1.0 : 0.5
             
-            UIView.animateWithDuration(0.30) {
+          UIView.animate(withDuration: 0.30) {
```

Comment:
Sure, But I fail to see the point of an empty animation block! Did you mean to put that alpha value change inside the animation blog?